TileTowns/Assets/Scripts/Buildings/Woodmill.cs

```diff
@@ -15,7 +15,7 @@ public class Woodmill : Building
         public override void UpdateMutation(ITileData tileData, IGameStateTurnMutation mutation)
         {
             int adjacentForests = CountAdjacentForestTiles(tileData);
-            mutation.BuildingIncome = moneyPerWorker * tileData.WorkersAssigned * (1 + adjacentForests);
```

Comment:
Writing this I wanted to slightly minimise the scalability. I feel like player is being punished too much whenever he doesn't maximise woodmill's output.

That could be justified that a tile is still a decent area of land so it has some trees too.

It's up to you tho

Reply:
I would say wood mill should be a building *for* utilizing trees, and nothing else. If you want to build something on a plane field without forest wood mill should not be a way to go

Reply:
And it won't be, surely farm will be significantly better withour surrounding trees

Reply:
to decide will add an issue